engine-precompiles/src/lib.rs

```diff
@@ -47,9 +47,9 @@ impl PrecompileOutput {
     }
 }
 
-impl From<PrecompileOutput> for evm::executor::PrecompileOutput {
+impl From<PrecompileOutput> for executor::stack::PrecompileOutput {
```

Comment:
These changes are unrelated to the core goal of this PR to properly handle reverts during contract deploy, but were needed after getting up to date with the latest upstream changes of SputnikVM (and getting up to date there was needed for this fix because I needed to make a small change to SputnikVM for this to work properly).